ui/css/itcss/tools/utilities.scss

```diff
@@ -1,37 +0,0 @@
-/*
-  Utility Classes
- */
-
-/* lib */
-.flex-column {
-  display: flex;
-  flex-direction: column;
-}
-
-.flex-row {
-  display: flex;
-  flex-direction: row;
-}
-
-.pointer {
-  cursor: pointer;
-}
```

Comment:
`pointer` is not used as a classname

![Screenshot 2025-02-07 at 2 58 48 PM](https://github.com/user-attachments/assets/07918e43-d59c-432b-87d3-cc5a9e73cd49)